package.json

```diff
@@ -1,39 +1,53 @@
 {
-  "name": "root",
-  "private": true,
-  "workspaces": [
-    "shared",
-    "packages/*",
-    "playground",
-    "tests",
-    "docs"
-  ],
   "devDependencies": {
+    "@nx/js": "20.6.4",
     "@typescript-eslint/eslint-plugin": "^5.5.0",
     "@typescript-eslint/parser": "^5.5.0",
+    "concurrently": "9.1.2",
     "eslint": "^8.22.0",
     "eslint-config-react-app": "^7.0.0",
     "eslint-plugin-import": "^2.31.0",
     "glob": "^10.3.10",
-    "lerna": "^5.4.0",
-    "patch-package": "^6.4.7",
-    "typescript": "^5.3.3"
+    "nx": "20.6.4",
+    "serve": "14.2.4",
+    "typescript": "^5.3.3",
+    "vitest": "^2.0.3",
+    "wait-on": "8.0.3"
   },
+  "name": "root",
+  "pnpm": {
+    "ignoredBuiltDependencies": [
+      "sharp",
+      "workerd"
+    ],
+    "onlyBuiltDependencies": [
+      "@parcel/watcher",
+      "@sentry/cli",
+      "canvas",
+      "esbuild",
+      "nx"
+    ]
+  },
+  "packageManager": "pnpm@10.7.0+sha512.6b865ad4b62a1d9842b61d674a393903b871d9244954f652b8842c2b553c72176b278f64c463e52d40fff8aba385c235c8c9ecf5cc7de4fd78b8bb6d49633ab6",
+  "private": true,
   "scripts": {
-    "start": "lerna run --stream --scope @blocknote/example-editor dev",
-    "gen": "lerna run --stream --scope @blocknote/dev-scripts gen",
-    "start:built": "npx serve playground/dist -c ../serve.json",
-    "build:clean": "npm run clean && npm run gen && npm run clean && lerna run --stream build",
-    "build": "lerna run --stream build",
-    "build:site": "npm run gen && lerna run --stream docs:build --concurrency 1",
-    "lint": "lerna run --stream lint",
-    "bootstrap": "lerna bootstrap --ci -- --force && patch-package",
-    "install-new-packages": "lerna bootstrap -- --force && patch-package",
-    "test": "lerna run --stream test",
-    "install-playwright": "cd tests && npx playwright install --with-deps",
-    "deploy": "lerna publish -- --access public",
-    "prepublishOnly": "npm run build && cp README.md packages/core/README.md && cp README.md packages/react/README.md",
+    "dev": "nx run @blocknote/example-editor:dev",
+    "dev:docs": "nx run docs:dev",
+    "build": "nx run-many --target=build",
+    "build:clean": "pnpm run clean && pnpm run gen && pnpm run clean && pnpm run build",
+    "build:site": "pnpm run gen && nx run docs:build:site",
+    "clean": "nx run-many --target=clean",
+    "deploy": "nx release --skip-publish",
+    "gen": "nx run @blocknote/dev-scripts:gen",
```

Comment:
Minor thing, but for whatever reason, the logs when generating are only dumped into the console after `gen:examples`/`gen:docs` run. This isn't a big deal, it's just `gen:docs` especially takes a minute to run and looks like it's stuck during that time.